src/eom-util.h

```diff
@@ -65,6 +65,9 @@ G_GNUC_INTERNAL
 void     eom_util_show_file_in_filemanager   (GFile *file,
                                               GtkWindow *toplevel);
 
+gboolean eom_util_dialog_page_scroll_event_cb   (GtkWidget  *notebook,
+                                                GdkEventScroll *event);
```

Comment:
Fix the indentation on the previous line.

Reply:
Training and re do and re do again are best ways for learning. :)